platform/mellanox/iproute2/Makefile

```diff
@@ -0,0 +1,29 @@
+SHELL = /bin/bash
+.ONESHELL:
+.SHELLFLAGS += -e
+
+MAIN_TARGET = iproute2-mlnx_$(IPROUTE2_MLNX_VERSION_FULL)_$(CONFIGURED_ARCH).deb
+
+$(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
+	# Remove any stale files
+	rm -rf iproute2*
+
+	# Download iproute2 from debian repository
+	dget http://deb.debian.org/debian/pool/main/i/iproute2/iproute2_$(IPROUTE2_MLNX_VERSION_FULL).dsc
+
+	pushd iproute2-$(IPROUTE2_MLNX_VERSION)
+
+	# Create git repository to apply patches with stg
+	git init
+	git add -f *
+	git commit -qm "initial commit"
+
+	# Apply Patches
+	stg init
+	stg import -s ../patch/series
+
+	# Build debian
+	dpkg-buildpackage -us -uc -b -j$(SONIC_CONFIG_MAKE_JOBS) --admindir $(SONIC_DPKG_ADMINDIR)
+
```

Comment:
Doesn't renaming required here?

Reply:
Package name is not given in compilation command, it is being determined by field "Package" in control file, see it [here](https://www.debian.org/doc/manuals/debian-faq/pkg-basics.en.html).
 After I changed it, I came to know that addition changes were needed, all of them are in 0002-Rename-iproute2-debian-package.patch